content/en/guides/get-started/commands.md

```diff
@@ -117,7 +117,7 @@ npm run start
 
 Nuxt.js gives you the ability to host your web application on any static hosting.
 
-To deploy a static generated site make sure you have `target: static` in your `nuxt.config.js`.(For Nuxt >= 2.13:)
+To deploy a static generated site make sure you have `target: 'static'` in your `nuxt.config.js`.(For Nuxt >= 2.13:)
```

Comment:
I think this wording should be fix: `(For Nuxt >= 2.13:)`

Reply:
you mean removing the colon? didnt notice that in the first place. can add it to my PR if you like.

Reply:
Would be nice :)

Reply:
done! :) and thx for building nuxt!!